MonkeyWrench/MonkeyWrench.csproj

```diff
@@ -62,7 +62,6 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AssemblyInfo.cs" />
```

Comment:
It's generated: https://github.com/mono/monkeywrench/blob/master/MonkeyWrench/Makefile#L2